packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -672,7 +673,8 @@ export async function activate(extensionContext: vscode.ExtensionContext) {
     SfdxWorkspaceChecker,
     WorkspaceContext,
     taskViewService,
-    telemetryService
+    telemetryService,
+    TelemetryService
```

Comment:
Only concern is whether or not the capitalization distinction is enough to differentiate it from the core instance.

Reply:
Yeah I think it's fine as is.  We could also put it under a property like 
```services: {
  TelemetryService
  // Service2,
  // Service3, 
}, ```

I don't hate the idea of having a separated group of things we had to add to support an external extension. 

either way...good to go.